### PR TITLE
#857: Fix the issue image is not visible in WYSIWYG.

### PR DESCRIPTION
**Pull request by fandm**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [x] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Issue

https://github.com/ymcatwincities/openy/issues/857

## Steps for review

- [ ] Open any node with Content field.
- [ ] Add image.
- [ ] Verify the image is visible in WYSIWYG.

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2368 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2368 |
| Installation process | http://install.openy-dev.ffwua.com/build2368/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2368/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2368/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2368/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2368/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2368/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2368/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2368/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2370 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2370 |
| Installation process | http://install.openy-dev.ffwua.com/build2370/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2370/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2370/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2370/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2370/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2370/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2370/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2370/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2371 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2371 |
| Installation process | http://install.openy-dev.ffwua.com/build2371/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2371/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2371/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2371/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2371/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2371/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2371/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2371/profiles/contrib/openy/build/reports/behat

File: modules/custom/openy_ckeditor/openy_ckeditor.module [hunk 1]
```diff
@@ -86,3 +87,23 @@ function openy_ckeditor_tokens($type, $tokens, array $data, array $options, Bubb
 function openy_ckeditor_page_attachments_alter(array &$attachments) {
   $attachments['#attached']['library'][] = 'openy_ckeditor/openy-ckeditor';
 }
+
+/**
+ * Implements hook_entity_embed_alter().
+ */
+function openy_ckeditor_entity_embed_alter(array &$build, EntityInterface $entity, array &$context) {
+  if (isset($context['data-widget']) && $context['data-widget'] == 'drupalentity') {
```
Comment (ddrozdik) [inline modules/custom/openy_ckeditor/openy_ckeditor.module]: please use `return early pattern` here

**Review by ddrozdik — CHANGES_REQUESTED** [review]

File: modules/custom/openy_ckeditor/openy_ckeditor.module (hunk 1 above)
Comment (fandm) [reply modules/custom/openy_ckeditor/openy_ckeditor.module]: Done

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2373 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2373 |
| Installation process | http://install.openy-dev.ffwua.com/build2373/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2373/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2373/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2373/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2373/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2373/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2373/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build2373/profiles/contrib/openy/build/reports/behat

**Review by ddrozdik — APPROVED** [review]

**Pull request merged**